### PR TITLE
feat: use devcard V2 and download the png image over stream

### PR DESCRIPTION
**Pull request by kopancek**
Change the URL of the devcard to v2, download png directly with the streaming API.

### CODE REVIEW

**Comment by kopancek** [comment]
Verified on my own profile that it works: https://github.com/kopancek/kopancek

**Review by nensidosari — APPROVED** [review]
LGTM 🚀 , though wait for @omBratteng approval to make sure it's all good. 🙏

File: README.md
Comment (omBratteng) [inline README.md:28]:
```suggestion
  - Can be found at [https://app.daily.dev/api/id](https://app.daily.dev/api/id)
```

Since the ID is the userID (I presume), we can just link to the ID page in the API?

File: package.json
Comment (omBratteng) [inline package.json:3]: Sorry, forgot that I created a workflow for creating a new release, so you can revert this change, and remove the build output as the workflow will automagically create it.

**Review by omBratteng — CHANGES_REQUESTED** [review]

File: package.json
Comment (kopancek) [reply package.json:3]: Ah, nice!

File: README.md
Comment (kopancek) [reply README.md:28]: I'd rather link to the page with the settings, as we did before. Otherwise it might be confusing to the user.

**Review by omBratteng — APPROVED** [review]

**Pull request merged**
